components/ArtifactShort.vue

```diff
@@ -267,6 +261,15 @@ export default {
         ? this.artifact.artifact_group.owner_id == this.userid
         : false
     },
+    isDraft() {
+      let hasPublicationAttr = (this.artifact.artifact_group.publications !== undefined)
+      hasPublicationAttr && this.artifact.artifact_group.publications.forEach(publication => {
+        if (publication.artifact_id == this.artifact.id) {
+          return false;
+        }
+      });
+      return hasPublicationAttr && this.artifact.artifact_group.publications.length ? true : false;
```

Comment:
Shouldn't this line just be a `return true;` ?  Seems like a given artifact is only a draft if is not already in the publications list (e.g. if it is not caught by the check on line 267).

Reply:
Yes you are right!